### PR TITLE
DPAD-786: Twitch: Update UCP UI

### PR DESCRIPTION
**Pull request by gundams-and-dragons**
This PR moves the jwplayer code into a directory, and creates a new directory and player for the twitch player.

### CODE REVIEW

File: src/twitch/players/shared/TwitchPlayerWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+`;
+
+const TwitchPlayerTargetWrapper = styled.div`
+	padding-top: 56.25%;
+	position: relative;
+	height: 0;
+`;
+
+interface WindowTwitch extends Window {
+	Twitch?: TwitchApi;
```
Comment (p-forfa) [inline src/twitch/players/shared/TwitchPlayerWrapper.tsx:22]: 👍

File: src/twitch/players/MobileArticleVideoPlayer.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+		if (hasPartnerSlot) {
+			return '75px';
+		}
+
+		return '55px';
+	};
+
+	return (
+		<PlayerWrapper playerName="twitch-mobile-article-video">
```
Comment (p-forfa) [inline src/twitch/players/MobileArticleVideoPlayer.tsx:62]: Does this use the current player tracking events we have for JW? We might need at least one event to signal that the twitch player was loaded.

File: src/twitch/players/DesktopArticleVideoPlayer.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	z-index: 2;
+`;
+
+const moveDownAnimation = (right: number, width: number) => keyframes`
+	from {
+		right: ${right}px;
```
Comment (AlburIvan) [inline src/twitch/players/DesktopArticleVideoPlayer.tsx:24]: Can these props be passed down as `undefined` from `placeholderRef.current?.getBoundingClientRect();` ?

**Review by p-forfa — APPROVED** [review]
We may want to update the version here. Other than that, looks good.

**Pull request merged**